website/components/Hero/typewriter.ts

```diff
@@ -1,58 +1,58 @@
-var TxtType = function (this: any, el: any, toRotate: any, period: any) {
-  this.toRotate = toRotate;
-  this.el = el;
-  this.loopNum = 0;
-  this.period = parseInt(period, 10) || 2000;
-  this.txt = "";
-  this.tick();
-  this.isDeleting = false;
-};
-
-TxtType.prototype.tick = function () {
-  var i = this.loopNum % this.toRotate.length;
-  var fullTxt = this.toRotate[i];
-
-  if (this.isDeleting) {
-    this.txt = fullTxt.substring(0, this.txt.length - 1);
-  } else {
-    this.txt = fullTxt.substring(0, this.txt.length + 1);
-  }
-
-  this.el.innerHTML = '<span class="wrap">' + this.txt + "</span>";
-
-  var that = this;
-  var delta = 200 - Math.random() * 100;
-
-  if (this.isDeleting) {
-    delta /= 2;
-  }
-
-  if (!this.isDeleting && this.txt === fullTxt) {
-    delta = this.period;
-    this.isDeleting = true;
-  } else if (this.isDeleting && this.txt === "") {
-    this.isDeleting = false;
-    this.loopNum++;
-    delta = 500;
-  }
-
-  setTimeout(function () {
-    that.tick();
-  }, delta);
-};
-
-window.onload = function () {
-  var elements = document.getElementsByClassName("typewrite");
-  for (var i = 0; i < elements.length; i++) {
-    var toRotate = elements[i].getAttribute("data-type");
-    var period = elements[i].getAttribute("data-period");
-    // if (toRotate) {
-    //   new TxtType(elements[i], JSON.parse(toRotate), period);
-    // }
-  }
-  // INJECT CSS
-  var css = document.createElement("style");
-  css.type = "text/css";
-  css.innerHTML = ".typewrite > .wrap { border-right: 0.08em solid #fff}";
-  document.body.appendChild(css);
-};
+// var TxtType = function (this: any, el: any, toRotate: any, period: any) {
```

Comment:
Let's not maintain commented code, unless it's required.

Reply:
This is the implemented typewriter functionality, but then jimmy said to remove it for now, we will add later
so, I have commented it, cause it's affecting on website performance
![typewriter-animation](https://github.com/canopas/bite-space/assets/123158106/e1be4ce9-c5a5-49a3-9907-97d2f78a1cb9)

Reply:
but it will remain in git history right? we can pull it back when required.